components/searchbar/index.jsx

```diff
@@ -0,0 +1,12 @@
+import './index.scss'
+import React from 'react'
+
+const SearchBar = () => {
```

Comment:
We have different imagination about this part. It will cause further discussion but it's fine for now 🙂